sql/updates/world/master/2021_02_03_00_world_remove_pet_carrion.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM spell_script_names WHERE ScriptName='spell_hun_pet_carrion_feeder';
```

Comment:
you actually updated that script.. so why remove it from db then? :D

Reply:
I thought I did :-O
I'll delete it again I guess